cluster/gce/gci/configure-helper.sh

```diff
@@ -1273,10 +1273,12 @@ EOF
 # Create the log file and set its properties.
 #
 # $1 is the file to create.
+# $2: the log owner uid to set for the log file.
+# $3: the log owner gid to set for the log file.
 function prepare-log-file {
   touch $1
   chmod 644 $1
-  chown "${LOG_OWNER_USER:-root}":"${LOG_OWNER_GROUP:-root}" $1
+  chown "${2:-${LOG_OWNER_USER:-root}}":"${3:-${LOG_OWNER_GROUP:-root}}" $1
```

Comment:
By integer id. When we set the user to run as in the [security context](https://github.com/kubernetes/kubernetes/blob/55859a60fe09c412e183c92ad265cf0d52fbe3d8/staging/src/k8s.io/api/core/v1/types.go#L2918) for the container manifest, it needs to be an integer. I think it would be a bit more maintainable with what we currently have to use the UID in both the security context and configure-helper, because otherwise we're going to have to modify the container image every time to change the user.